packages/edit-site/src/components/add-new-template/utils.js

```diff
@@ -622,41 +624,57 @@ const useTemplatesToExclude = (
 const useEntitiesInfo = (
 	entityName,
 	templatePrefixes,
-	additionalQueryParameters = {}
+	additionalQueryParameters = EMPTY_OBJECT
 ) => {
```

Comment:
This was throwing a React warning because the useSelect was re-rendering the components while the data didn't change.

I just refactored it a bit to solve the above, but to be honest these hooks are very unclear, I'm not even sure we need them and why they exist as helpers. They just make things hard to follow.